api-tests/src/clients/core-back-internal-client.ts

```diff
@@ -37,12 +37,14 @@ export const initialiseIpvSession = async (
 export const sendJourneyEvent = async (
   event: string,
   ipvSessionId: string,
+  featureSet: string | undefined,
```

Comment:
Strictly speaking you should include the featureSet on all these API calls (not just journey event) - in case there's any behaviour that changes in the other endpoints

Reply:
Good point, will do